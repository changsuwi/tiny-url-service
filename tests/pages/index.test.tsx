import { render, screen } from '@testing-library/react';
import Home from '../../pages/index';

describe('Render Home Component', () => {
  const { container } = render(<Home />);
  it('renders a URL text box', () => {
    const textbox = screen.getByRole('textbox');

    expect(textbox).toBeInTheDocument();
  });
});
