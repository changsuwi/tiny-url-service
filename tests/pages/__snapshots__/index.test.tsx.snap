// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Home Component renders a URL text box 1`] = `
<input
  name="Original URL Textbox"
  type="text"
  value=""
/>
`;
